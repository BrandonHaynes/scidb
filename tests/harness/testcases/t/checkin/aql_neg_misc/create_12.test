--test
--error --code=scidb::SCIDB_SE_SYNTAX::SCIDB_LE_INCORRECT_DIMENSION_BOUNDARY --aql="create array A <a:int32> [x=-9223372036854775807:0,5,0]"
