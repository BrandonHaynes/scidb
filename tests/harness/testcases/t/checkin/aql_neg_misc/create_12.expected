[An error expected at this place for the query "create array A <a:int32> [x=-9223372036854775807:0,5,0]". And it failed with error code = scidb::SCIDB_SE_SYNTAX::SCIDB_LE_INCORRECT_DIMENSION_BOUNDARY. Expected error code = scidb::SCIDB_SE_SYNTAX::SCIDB_LE_INCORRECT_DIMENSION_BOUNDARY.]

